### PR TITLE
Add prereq install instructions for mac

### PR DESCRIPTION
**Pull request by wgwz**
## Description

This PR adds some prerequisite steps that I needed for `yarn install` to run successfully, during local development on my mac. I believe I captured all the prerequisites, though it is possible I missed some (i.e. `make` is another dependency but I already had that installed).

It addresses these issues that I was getting when running `yarn install`:
- https://gist.github.com/wgwz/d560c9a5541491100b8ae3ac5153f24e
- https://gist.github.com/wgwz/76db41b56bcb8c0f60ccf90f9c6914e0

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by blushi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,7 +31,20 @@ This project uses [lerna](https://github.com/lerna/lerna) with [yarn workspaces]
 - `web-www`: Regen website based on [Gatsby](https://www.gatsbyjs.org/)
 - `web-auth`: React application used for Auth0 Custom Universal Login
 
-Install dependencies using:
+### Prerequisites
+
+#### Mac
+
+To set up system dependencies run the following:
```
Comment (blushi) [inline README.md:38]: maybe good to add that this is only needed if you don't already have python and/or vips installed

**Review by haveanicedavid — APPROVED** [review]
Thanks for adding this!

File: README.md [hunk 2]
```diff
@@ -31,7 +31,20 @@ This project uses [lerna](https://github.com/lerna/lerna) with [yarn workspaces]
 - `web-www`: Regen website based on [Gatsby](https://www.gatsbyjs.org/)
 - `web-auth`: React application used for Auth0 Custom Universal Login
 
-Install dependencies using:
+### Prerequisites
+
+#### Mac
+
+If you haven't already, you can set up system dependencies run the following:
```
Comment (blushi) [inline README.md:38]:
```suggestion
If you haven't already, you can set up system dependencies by running the following commands: 
```

**Comment by blushi** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**